Amasty_GiftCard/templates/checkout/cart/giftcard.phtml

```diff
@@ -118,27 +121,31 @@
                             name="am_giftcard_code"
                             value=""
                             class="input__field cart-discount__content-field"
-                            placeholder="<?= $block->escapeHtmlAttr(__('Enter gift card code')) ?>"
+                            placeholder="<?= /* @noEscape */ $block->escapeHtmlAttr($enterGiftCardLabel) ?>"
                             data-validate="{required:true}"
                         >
                     </div>
-
+                    <?php
+                        $addGiftCartLabel = __('Add Gift Card');
+                    ?>
                     <button
                         class="button cart-discount__content-button"
                         type="submit"
-                        value="<?= $block->escapeHtmlAttr(__('Add Gift Card')) ?>"
+                        value="<?= /* @noEscape */ $block->escapeHtmlAttr($addGiftCartLabel) ?>"
```

Comment:
we use either noEscape comment f data doesn't need to be escaped or some escape method. Here we use properly escapeHtmlATtr and this is ok